modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLSelectFolderDisplayContext.java

```diff
@@ -318,6 +321,12 @@ private boolean _isMountFolderVisible() {
 	private static final Log _log = LogFactoryUtil.getLog(
 		DLSelectFolderDisplayContext.class);
 
+	private static final Snapshot<ModelResourcePermission<Folder>>
+		_folderModelResourcePermissionSnapshot = new Snapshot<>(
+			DLSelectFolderDisplayContext.class,
+			Snapshot.cast(ModelResourcePermission.class),
+			"(model.class.name=" + Folder.class.getName() + ")");
+
```

Comment:
@W2jx, `new DLSelectFolderDisplayContext(...) `only happened in DLFolderItemSelectorView.class and DLFolderItemSelectorView is a @Component, so you may reference this in DLFolderItemSelectorView as the below and then make _dlFolderModelResourcePermission as constructor param for DLSelectFolderDisplayContext used.
```
@Reference(
		target = "(model.class.name=com.liferay.document.library.kernel.model.DLFolder)"
	)
	private ModelResourcePermission<DLFolder> _dlFolderModelResourcePermission;
```